Source/Core/Core/ConfigManager.cpp

```diff
@@ -806,6 +806,9 @@ DiscIO::Region SConfig::ToGameCubeRegion(DiscIO::Region region)
 
 const char* SConfig::GetDirectoryForRegion(DiscIO::Region region)
 {
+  if (region == DiscIO::Region::Unknown)
+    region = GetFallbackRegion();
```

Comment:
You need a call to ToGameCubeRegion around this call to GetFallbackRegion, otherwise we can run into trouble with NTSC-K Wii Menus.

Reply:
Oh yeah you're right. Good catch.